plugin/arcconf/arcconf.py

```diff
@@ -484,14 +484,11 @@ def volumes(self, search_key=None, search_value=None,
     def _arcconf_disk_to_lsm_disk(arcconf_disk, sys_id, ctrl_num):
         disk_id = arcconf_disk['serialNumber'].strip()
 
-        disk_name = "%s" % (arcconf_disk['model'])
+        disk_name = "%s - %s" % (arcconf_disk['vendor'], arcconf_disk['model'])
         disk_type = _disk_type_of(arcconf_disk)
-        link_type = disk_type
+        link_type = _disk_link_type_of(arcconf_disk)
 
-        try:
-            blk_size = int(arcconf_disk['physicalBlockSize'])
-        except KeyError:
-            blk_size = int(arcconf_disk['blockSize'])
+        blk_size = int(arcconf_disk['blockSize'])
```

Comment:
@sharath-t Can you explain to me why `physicalBlockSize` is not used here?

Reply:
Thanks for the catch.
i discussed internally and found that 'physicalBlockSize' was added to support 4kn drives. so i have reverted the code to earlier state.